### ISSUE TITLE
Optimize the limit of descendants

### ISSUE THREAD

**Issue by ReidyT** [graasp/graasp#1494]
This issue will be divided into sub-issues, but the ultimate aim is to increase the limitation on the number of descendants when copying/moving/importing and creating elements.

To do this, we need to complete several steps before we can begin optimising the code.

# 1. Understanding our current limits

First of all, you need to be aware of the system's limitations. To do this, we will use loading tests.

- [ ] Find limits for the number of requests to obtain the home screen so that delays are acceptable
- [ ] Find limits for the number of descendants when copying items so that delays for other users (getting home screen) are acceptable
- [ ] Find other limits but in another PR

# 2. Understanding the bottleneck of limitation

Once we've found the limits on the number of descendants, we need to investigate and understand why we have this limit.

# 3. Optimise the code

We can now modify and optimise the code to increase the maximal number of descendants.

# 4. Validate the optimisation

We need to run the load tests again to validate that our optimisation is working properly, and to define new limits if there are any unknowns.
